### PR TITLE
CTDA-1271 added security token and notification url field definitions

### PR DESCRIPTION
**Pull request by RobertLaverick**

### CODE REVIEW

**Review by crystalmark — APPROVED** [review]

File: resources/public/api/definition.json [hunk 1]
```diff
@@ -18,7 +18,18 @@
         "endpointsEnabled": true,
         "access": {
           "type": "PUBLIC"
-        }
+        },
+        "fieldDefinitions": [{
+          "name": "notificationUrl",
+          "description": "What is your notification web address for us to send push notifications to?",
+          "type": "URL",
```
Comment (DavidGregory084) [inline resources/public/api/definition.json:25]: This should be a PPNSField

File: resources/public/api/definition.json [hunk 2]
```diff
@@ -18,7 +18,18 @@
...
+          "hint": "You must only give us a web address that you own. Your application will use this address to listen to notifications from HMRC."
+        }, {
```
Comment (DavidGregory084) [inline resources/public/api/definition.json:27]: I think there should be validation rules for this field similar to https://github.com/hmrc/goods-movement-system-haulier-api/blob/master/app/uk/gov/hmrc/goodsmovementsystemhaulierapi/platform/views/definition.scala.txt#L30

**Review by DavidGregory084 — CHANGES_REQUESTED** [review]

File: resources/public/api/definition.json (hunk 1 above)
Comment (DavidGregory084) [reply resources/public/api/definition.json:25]: Again see e.g. https://github.com/hmrc/goods-movement-system-haulier-api/blob/master/app/uk/gov/hmrc/goodsmovementsystemhaulierapi/platform/views/definition.scala.txt#L30

**Review by DavidGregory084 — APPROVED** [review]
Looks good to me 💯

**Pull request merged**